src/plugins/intel_cpu/src/nodes/col2im.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "col2im.h"
+#include "common/cpu_convert.h"
+#include "openvino/reference/col2im.hpp"
+#include "openvino/op/col2im.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+Col2Im::Col2Im(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr context)
+    : Node(op, context, NgraphShapeInferFactory(op, EMPTY_PORT_MASK)) {
+    const auto col2Im = ov::as_type_ptr<const ov::op::v15::Col2Im>(op);
+    strides = col2Im->get_strides();
+    dilations = col2Im->get_dilations();
+    padsBegin = col2Im->get_pads_begin();
+    padsEnd = col2Im->get_pads_end();
+}
+
+bool Col2Im::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        const auto col2Im = std::dynamic_pointer_cast<const ov::op::v15::Col2Im>(op);
+        if (!col2Im) {
+            errorMessage = "Only opset15 Col2Im operation is supported";
+            return false;
+        }
+    } catch (...) {
+        return false;
+    }
+    return true;
+}
+
+void Col2Im::getSupportedDescriptors() {
+    // Validation is already done in the ov::opset15::Col2Im.
+}
+
+void Col2Im::initSupportedPrimitiveDescriptors() {
+    if (!supportedPrimitiveDescriptors.empty())
+        return;
+
+    ov::element::Type dataPrecision = getOriginalInputPrecisionAtPort(0);
+    ov::element::Type indexPrecision = getOriginalInputPrecisionAtPort(1);
+    ov::element::Type outputPrecision = getOriginalOutputPrecisionAtPort(0);
```

Comment:
`outputPrecision` is excessive, due to it's equal to `dataPrecision`

Reply:
Removed.